packages/edit-site/src/components/global-styles/screen-root.js

```diff
@@ -52,6 +71,8 @@ function ScreenRoot() {
 
 	return (
 		<Card size="small" className="edit-site-global-styles-screen-root">
+			<ScreenRootSlot />
```

Comment:
This is a too prominent place to add a slot and plugins will compete too much for place. I think the preview and variations should stay at the top. Let's move this slot after the "browse styles" button